src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -83,6 +93,11 @@ private static void OnProcessExit(object sender, EventArgs e)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
 
+            if (_logger.IsDefaultPlatformLoggingEnabled)
+            {
+                s_lazyCore.Value.LogEvent += LogEventRaised;
```

Comment:
This is an event handle leak. You keep adding handles, but you never remove them. 

I think you need to register the callback on every IBroker method and remove it in a "finally" block.

Reply:
Done. In a different way.